db/commitlog/commitlog.cc

```diff
@@ -205,8 +205,19 @@ class db::commitlog::segment_manager : public ::enable_shared_from_this<segment_
     bool _shutdown = false;
     std::optional<shared_promise<>> _shutdown_promise = {};
 
+    struct request_controller_timeout_exception_factory {
+        class request_controller_timed_out_error : public timed_out_error {
+        public:
+            const char* what() const noexcept override {
```

Comment:
We prepend overrides of virtual methods with `virtual` keyword.

Reply:
OK, but why?

Reply:
Because you only need to read one word to see that this is a virtual method vs. having to read all of them to the end.

Reply:
There are people among us who read in opposite direction :)
Anyway, done